docs/advanced-scheduled-price-changes/README.md

```diff
@@ -1,8 +1,8 @@
-# Scheduled Price Changes
+# ADVANCED: Scheduled Price Changes
 
-Tags: Price, Products, Schedule
+Tags: Advanced, Price, Products, Schedule
```

Comment:
Can you add tags: sale, discount

^^undestanding this isn't quite a discount, but I think it'll help with task discoverability